tools/c7n_mailer/c7n_mailer/email_delivery.py

```diff
@@ -175,6 +175,10 @@ def get_email_to_addrs_to_resources_map(self, sqs_message):
         # or it's an email from an aws event username from an ldap_lookup
         email_to_addrs_to_resources_map = {}
         targets = sqs_message['action']['to']
+
+        if sqs_message['action'].get('cc'):
```

Comment:
this isn't sufficient to add cc support, you'll need to add the cc attribute on the email message and attach the cc emails to that attribute.

Reply:
To properly implement cc'ing, we'll need to modify how the address to resource map stores each address, so we can distinguish which are 'to' addresses and which are proper 'cc's. I feel like this speaks to a greater need for refactoring some of the mailer, though that would be a much larger effort.